packages/components/stories/DateTimePicker.js

```diff
@@ -2,37 +2,16 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { IconsProvider } from '../src/index';
 
-import DateTimePicker, {
-	DateTimeView,
-	MonthYearView,
-} from '../src/DateTimePickers';
+import DateTimePicker from '../src/DateTimePickers';
 
 
 storiesOf('DateTimePicker', module)
 	.add('Full DateTimePicker structure', () => (
 		<div>
 			<h1>DateTimePicker structure</h1>
 			<IconsProvider />
-			<div style={{ width: '320px' }}>
+			<div style={{ width: '320px', border: '1px solid black' }}>
```

Comment:
why do you need this hard coded style ?

Reply:
The final bordure style will be implemented by a wrapper component like the future "DateTimePickerDropdown" or something equivalent.
It's now only to know where the component start and end.

Reply:
✔